pkg/reconciler/instances/scmigration/resource_cleanup.go

```diff
@@ -0,0 +1,139 @@
+package scmigration
+
+import (
+	"k8s.io/apimachinery/pkg/apis/meta/v1/unstructured"
+)
+
+const (
+	HelmBrokerComponent           = "helm-broker"
+	ServiceCatalogAddonsComponent = "service-catalog-addons"
+	ServiceCatalogComponent       = "service-catalog"
+	ServiceManagerProxyComponent  = "service-manager-proxy"
+)
+
+var (
+	resources = map[string][]*unstructured.Unstructured{
+		HelmBrokerComponent:           hbResources,
+		ServiceCatalogAddonsComponent: svcatAddonsResources,
+		ServiceCatalogComponent:       svcatResources,
+		ServiceManagerProxyComponent:  smProxyResources,
+	}
+
+	smProxyResources = []*unstructured.Unstructured{
```

Comment:
all of the resources in [helm_broker.go](https://github.com/kyma-incubator/reconciler/blob/e58425860a8841e4442cd70b8f4eebd8d1c42039/pkg/reconciler/instances/scmigration/helm_broker.go), [service_catalog.go](https://github.com/kyma-incubator/reconciler/blob/e58425860a8841e4442cd70b8f4eebd8d1c42039/pkg/reconciler/instances/scmigration/service_catalog.go), [service_catalog_addons.go](https://github.com/kyma-incubator/reconciler/blob/e58425860a8841e4442cd70b8f4eebd8d1c42039/pkg/reconciler/instances/scmigration/service_catalog_addons.go) and [service_manager_proxy.go](https://github.com/kyma-incubator/reconciler/blob/e58425860a8841e4442cd70b8f4eebd8d1c42039/pkg/reconciler/instances/scmigration/service_manager_proxy.go) here were generated from the helm release YAML files directly by https://github.com/wozniakjan/reverse-kube-resource

This file just aggregates the generated files for convenience.